concent_api/core/migrations/0014_storedmessage_protocol_version.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
What about data migration?

Reply:
I don't understand.. migration is generated

Reply:
Ok I see you have set the default to `2.15.0`, which should work.